_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,320 @@
+
+**Note:** The test targets are visible link elements that are part of a block of content that is displayed inline.
+
+## Expectation
+
+For each test target, the [outcome][] of one of the following rules is "passed":
+
+- [Inline link has distinguishable content](https://act-rules.github.io/rules/e7cc2b)
+- [Inline link has distinguishable style not based on color alone](https://act-rules.github.io/rules/88407d)
+- [Inline link has distinguishable border](https://act-rules.github.io/rules/36f116)
+- [Inline link has distinguishable box-shadow](https://act-rules.github.io/rules/66e9f0)
+- [Inline link has different foreground color and distinguishable style on hover and focus](https://act-rules.github.io/rules/548868)
+- [Inline link has different background color and distinguishable style on hover and focus](https://act-rules.github.io/rules/2803b8)
+
+## Assumptions
+
+- This rule assumes that [`br` HTML elements][br] are only used for line breaks that are actually part of the content and not for separating thematic groups of the content. 
+- This rule assumes that the link is distinguishable from the rest of the text with color, which means it fails SC 1.4.1 when there is not another way to distinguish it.
+- This rule assumes that the 3:1 contrast difference between text is minimal to what would be sufficient to meet WCAG 2.0. This value is part of [technique G183](https://www.w3.org/WAI/WCAG21/Techniques/general/G183), but is not specified in the [1.4.1 success criterion](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html).
+- This rule assumes that any change in font is sufficiently distinguishable, and that fonts are loaded when they are present.
+- This rule assumes that if multiple colors are used in the visible text nodes of the _ancestor_ element then color can not be a distinguishing factor.
+- This rule assumes that if `box-shadow` is used in the different visible text nodes of the _ancestor_ element then `box-shadow` can not be a distinguishing factor.
+- This rule assumes that if `border` is used in the different visible text nodes of the _ancestor_ element then `border` can not be a distinguishing factor.
+
+## Accessibility Support
+
+_No accessibility support issues known._
+
+## Background
+
+- [Understanding Success Criterion 1.4.1: Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)
+- [G183: Using a contrast ratio of 3:1 with surrounding text and providing additional visual cues on focus for links or controls where color alone is used to identify them](https://www.w3.org/WAI/WCAG21/Techniques/general/G183)
+- [F73: Failure of Success Criterion 1.4.1 due to creating links that are not visually evident without color vision](https://www.w3.org/WAI/WCAG21/Techniques/failures/F73)
+- [C15: Using CSS to change the presentation of a user interface component when it receives focus](https://www.w3.org/WAI/WCAG21/Techniques/css/C15)
+
+## Test Cases
+
+### Passed
```

Comment:
@WilcoFiers I'm not sure I agree it should pass. There is nothing in these links, apart from color, to distinguish them from the text (the slashes). This results in a sighted user (probably most sighted users, not just colorblind) failing to understand if there is one, three or four links in this paragraph. On the other blind, a blind user will have no doubt that there are three links in there.

@ShadowBB what is your take on this?

Reply:
Although Carlos has a point, that this could still cause accessibility issues, I don't think we can truly let this example fail for this particular rule because the fact that it is placed in this menu like style communicates visually to the user that these are links.

However you could argue that this example already passes, because it would pass rule "e7cc2b" because the content (in this case slashes) makes it visually apparent that these are links.

Reply:
After reading f73 again, I am am not so sure about what I said here on 29-April.

It seems that a menu like this:

```
<style> a { text-decoration:none }</style>
<p>
  <a href="//W3.org/">W3C</a> / 
  <a href="//W3.org/WAI/">WAI</a> / 
  <a href="//W3.org/WAI/standards-guidelines/">Standards / Guidelines</a>
</p>
```

Should fail because the links in there are not "visually identifiable via some other means" in any objective way.

Reply:
@WilcoFiers I would fail this because color is the only means of conveying what is a link. You cannot use the slash as a distinguishing feature because you don't have 4 links, which is implied by the 3 slashes. Remove the color information and tell me how many links are there.

Reply:
Having re-read this thread, I admit I was looking at this from a wrong perspective. Every link in there has something that indicates it is a link (the slash adjacent to it). I was concerned with the last entry that should have been split in two links (because of the slash), but that is irrelevant to this rule.

On a positive note, that is a pass, given that, as you say, there is distinguishing content indicating that it is a link!

Reply:
No longer relevant, given that we dropped the expectation about content being used to distinguish links.